### ISSUE TITLE
Weird JSON example after migration from 5.5.x to 5.6.x

### ISSUE THREAD

**Issue by epot** [domaindrivendev/Swashbuckle#1137]
I have the example that looks like this after migrating:
{
  "application/json": {
    "id": "a0be8794-515f-4a12-bef7-f27bb67ff4f8",
    "userId": "500d4851-7a4c-41b1-9389-831cccec45f3",
    ...
  }
}

The additional "application/json" was not there before. Is it normal?

**Comment by furier** [comment]
We had the same issue, and changed our `ExampleResponseOperationFilter` to not set the `response.examples` property to a dictionary of `{ { "application/json": {...} } }` to the object within the `application/json` key.

That fixed the examples being displayed as the whole dictionary rather then the example for the appropriate response type selected in the swagger ui.

However it seems to completely break the swagger specifications as the `response.examples` attribute should be a dictionary, making swagger editor complain that the examples property should be set to an object. _(even tho that doesn't make much sense to me)_

AutoRest is as a result gives an error when generating the client from the swagger specifications.

**Comment by ntregillus** [comment]
yeah, the standard format for json per the open-api standard is `contenttype : { object def here }`  when I use the editor (https://editor.swagger.io/) it displays correctly. its just swashbuckle's implementation of swagger ui

**Comment by lorenh** [comment]
Anyone have a solution to this issue?
